src/style.scss

```diff
@@ -0,0 +1,130 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto:regular,500,700&display=swap");
+
+:root {
+  --сolor-white: rgb(255, 255, 255);
+  --color-blue: rgba(0, 172, 220, 1);
+  --color-gray: rgba(97, 96, 112, 1);
+  --color-border: rgba(243, 243, 243, 1);
+}
+
+body {
+  font-family: "Roboto", sans-serif ;
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  border-radius: 5px;
+  padding: 0 16px;
+  width: 200px;
+  border: 1px solid var(--color-border);
+
+  .card__img {
+    margin: 0 auto;
+    margin-top: 32px;
+    margin-bottom: 40px;
+    max-width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    font-family: inherit;
+    color: rgba(6, 11, 53, 1);
```

Comment:
And one more :)
![image](https://github.com/mate-academy/layout_product-cards/assets/86923089/1f872962-40e3-40ad-b511-80e83a372eab)

Please, do not use RGBA if you don't need it (4th parameter is 1, so it doesn't make any sense). Try to stick 1 type of color (better HEX). If you have repeatable colors - move them to variables, and use these variables.